app/src/main/java/org/wikipedia/dataclient/page/PageSummary.java

```diff
@@ -31,6 +31,7 @@
     @Nullable private Thumbnail thumbnail;
     @Nullable @SerializedName("originalimage") private Thumbnail originalImage;
     @Nullable private String lang;
+    @Nullable private String revision;
```

Comment:
Shouldn't this be a `long` instead of a `String`? (and everywhere else?)

Reply:
make sense, done!